### ISSUE TITLE
[Feature Request] Autocomplete highlight text color SASS variable

### ISSUE THREAD

**Issue by megaroeny** [vuetifyjs/vuetify#12234]
### Problem to solve
To summarize, color contrast.

Currently, the text color cannot be changed for highlighting (`v-list-item_mask`) the text from an autocomplete search. My concern is the color contrast. So it would be nice if another SASS variable was added so we can control the color for the text itself being highlighted.

### Proposed solution
Add another variable, something like:
```
'expansion-panels': (
    'focus-text':
)
```

**Note:** The fact that `expansion-panels` is being used for this is super confusing. Took me a while to figure out how/why lol...

**Comment by mrc-bsllt** [comment]
Is there a solution to achieve this?
